### ISSUE TITLE
Style of the slickgrid interfere the global UL/LI bullet style.

### ISSUE THREAD

**Issue by boogiebug** [ghiscoding/slickgrid-universal#868]
### Describe the bug

I'm using Angular with slickgrid-universal and found that according to the following line. It cause all bullet  in front of the UL/LI tag in the whole app disappear.

```scss
ul {
  list-style-type: none;
}
```

https://github.com/ghiscoding/slickgrid-universal/blob/master/packages/common/src/styles/slick-bootstrap.scss#:~:text=ul%20%7B,%7D

The most painful point is it is not easily overridable, since the selector is at the most global level. My current only work-around is to put more specific selector to other UL/LI, which is is need to do with all other components.

My suggestion is adding more selector or move it into class .slickgrid-container, to make it specific to the slickgrid only.

### Reproduction

Simply adding the slickgrid universal stylesheet into your project, and this will happend.

### Which Framework are you using?

Angular

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| (framework used)    | VERSION |
| Slickgrid-Universal | VERSION |
| TypeScript          | VERSION |
| Browser(s)          | VERSION |
| System OS           | VERSION |
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/slickgrid-universal/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [Wikis](https://github.com/ghiscoding/slickgrid-universal/wiki).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/slickgrid-universal/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/slickgrid-universal/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
oh I see, it seems that my change in this [PR](https://github.com/ghiscoding/slickgrid-universal/pull/488/files) was a little too aggressive on the `slick.bootstrap.scss` file. It could be fixed but only for latest Slickgrid-Universal 2.x and Angular-Slickgrid 5.x, I'm not sure which versions you are since you didn't provide that info above. Feel free to submit a PR if you have time, simply removing this code

https://github.com/ghiscoding/slickgrid-universal/blob/cd2cb16d9cd1e50b5a86d1c0103681494d18384f/packages/common/src/styles/slick-bootstrap.scss#L24-L26

should be enough to fix the issue, from that PR #488, it looks like I already have necessary bullet removal on other part of the grid (mainly pagination) that were a problem in the past.

**Comment by ghiscoding** [comment]
The fix was released in Angular-Slickgrid [v5.3.0](https://github.com/ghiscoding/Angular-Slickgrid/releases/tag/v5.3.0) and Slickgrid-Universal [v2.3.0](https://github.com/ghiscoding/slickgrid-universal/releases/tag/v2.3.0)
